Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:2
+
+RUN set -x \
+    && apt-get update -qq \
+    && apt-get install -y -qq --no-install-recommends gcc git musl-dev python-dnspython python-xdg \
+    && pip install --no-cache-dir git+https://github.com/linkcheck/linkchecker.git@master \
+    && apt-get -y -qq purge gcc git musl-dev \
+    && apt-get autoremove -y -qq \
+    && apt-get clean -y -qq all \
+    && rm -rf /var/lib/apt/lists/* /usr/share/doc /usr/share/locale /usr/share/man
+
+VOLUME /mnt
+
+WORKDIR /mnt
```

Comment:
not sure why those are necessary?

Reply:
This will be used in case I would like to test "local files" using `linkchecker`.
I'll update documentation once we the `Dockerfile` will be merged to have it documented properly...

Reply:
ah okay, that makes sense... maybe add a comment explaining that? :)

Reply:
Done...